config/uitests-amo.js

```diff
@@ -0,0 +1,19 @@
+// For use on addons-server, or with the command `amo:ui-tests` and the
+// HOSTNAME  variable as `ui-tests`.
+module.exports = {
+  apiHost: 'http://olympia.test',
+  CSP: false,
```

Comment:
Can you put this line after `proxyEnabled` and add one empty line before and after? I'd like to make it super easy to see that `CSP` is disabled.